pkg/app/master/commands/cliflags.go

```diff
@@ -37,6 +38,7 @@ const (
 	FlagLogLevelUsage      = "set the logging level ('trace', 'debug', 'info', 'warn' (default), 'error', 'fatal', 'panic')"
 	FlagLogUsage           = "log file to store logs"
 	FlagLogFormatUsage     = "set the format used by logs ('text' (default), or 'json')"
+	FlagConsoleOutputUsage = "set the format used by logs ('text-output' (default), or 'json-output')"
```

Comment:
we can drop the `-output` suffix, so it's just `text` or `json`

Reply:
Done, for now using json as default just to check but later on switch to text agan